docs/operations/images.md

```diff
@@ -119,54 +135,33 @@ aws ec2 describe-images --region us-east-1 --output table \
   --filters "Name=name,Values=debian-10-amd64-*"
 ```
 
-## Ubuntu 18.04 (Bionic)
-
-Ubuntu 18.04 is based on Kernel version **4.15** which has number of known bugs that affect it and may be noticed with larger clusters:
-
-* [kubernetes/kubernetes#56903](https://github.com/kubernetes/kubernetes/issues/56903)
-* [kubernetes/kubernetes#67577](https://github.com/kubernetes/kubernetes/issues/67577)
-
-Available images can be listed using:
-
-```bash
-aws ec2 describe-images --region us-east-1 --output table \
-  --owners 099720109477 \
-  --query "sort_by(Images, &CreationDate)[*].[CreationDate,Name,ImageId]" \
-  --filters "Name=name,Values=ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-*"
-```
-
-## Ubuntu 20.04 (Focal)
+### Flatcar
 
-Ubuntu 20.04 is based on Kernel version **5.4** which fixes all the known major Kernel bugs.
+Flatcar is a friendly fork of CoreOS and as such, compatible with it.
 
 Available images can be listed using:
 
 ```bash
 aws ec2 describe-images --region us-east-1 --output table \
-  --owners 099720109477 \
+  --owners 075585003325 \
   --query "sort_by(Images, &CreationDate)[*].[CreationDate,Name,ImageId]" \
-  --filters "Name=name,Values=ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-*"
+  --filters "Name=name,Values=Flatcar-stable-*-hvm"
 ```
 
-## CentOS 7
-
-CentOS 7 is based on Kernel version **3.10** which has a considerable number of known bugs that affect it and may be noticed in production clusters:
-
-* [kubernetes/kubernetes#56903](https://github.com/kubernetes/kubernetes/issues/56903)
-* [kubernetes/kubernetes#67577](https://github.com/kubernetes/kubernetes/issues/67577)
+### Kope.io
```

Comment:
Should go into the "Deprecated Distros" section.

Reply:
True. Done.
I think, based on the Debian 9 moving to LTS, we should deprecate it too soon (but not remove it). Just show people it's a bad idea to use it.